packages/example-getting-started/docs/1-prerequisites-and-setup.md

```diff
@@ -0,0 +1,30 @@
+## Prerequisites
```

Comment:
Can we signify that if they are to follow the instructions here, they get the fully functional app, and if they follow the other steps, they're following  a tutorial to build it?

Reply:
See https://github.com/strongloop/loopback-next/pull/1001#discussion_r168594338